ui/packages/lib/src/components/accordion_form/AccordionForm.js

```diff
@@ -23,70 +23,63 @@ export const AccordionForm = ({
   const { height: lastSectionHeight } = useDimension(lastSectionRef);
 
   return (
-    <StickyContainer className="container">
-      <EuiFlexGroup direction="row" gutterSize="none" className="accordionForm">
-        <EuiFlexItem grow={false} className="accordionForm--sideNavContainer">
-          <Sticky>
-            {({ style, isSticky }) => (
-              <span style={style}>
-                {isSticky && <EuiSpacer size="m" />}
-                <AccordionFormSideNav name={name} sections={sections} />
-              </span>
-            )}
-          </Sticky>
```

Comment:
This file looks super different because the indentation has changed slightly with the removal of the overarching `StickyContainer` object. The main change basically involves the that, the replacing of the `Sticky` block with the one commented below.